tasks/sep/032-op-pectra-defence/README.md

```diff
@@ -0,0 +1,44 @@
+# ProxyAdminOwner - Set Dispute Game Implementation
+
+Status: READY TO SIGN
+
+## Objective
+
+This task updates deploys new dispute games with upgrade prestate hashes on OP Sepolia and Sony Minato.
+
+This updated prestate hash corresponds with the hardfork to support the L1 Pectra upgrade.
+
+## Input data derivation:
+
+The OPPrestateUpdater contract was deployed in advance to [0x86b5BFD11625FaaE6Ed5f4f51786C2C0C3359411](https://sepolia.etherscan.io/address/0x86b5BFD11625FaaE6Ed5f4f51786C2C0C3359411).
```

Comment:
TODO: this will need to be redeployed, as the [Deploy script](https://github.com/ethereum-optimism/optimism/blob/f45e785bf9bf6424cf0efa981a0cc75739467321/packages/contracts-bedrock/scripts/deploy/DeployOPPrestateUpdater.s.sol#L51-L52) read the blueprints from the old proxied OPCM, not the updated unproxied one (see this PR for [correct addresses](https://github.com/ethereum-optimism/superchain-registry/pull/919/files#diff-92de748a6cadfaee21bd25cc20434f7e1eb97472b517db8dbb53f42acda5e1cd))

Reply:
I did that
506bdc63f1c1143e3506532f2ed05a39dca515a0